src/shared/env-file-label.c

```diff
@@ -20,14 +20,14 @@ int write_env_file_label(int dir_fd, const char *fname, char **headers, char **l
         return r;
 }
 
-int write_vconsole_conf_label(char **l) {
+int write_vconsole_conf_label(const char* vconsole_conf_path, char **l) {
```

Comment:
It is not necessary to take the path.
Please call `etc_vconsole_conf()` in this function.